src/main/java/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,14 @@
+package calculator
+
+import calculator.extractor.TextDelimiterExtractorFactory
+
+data class StringCalculator(
+    private val factory: TextDelimiterExtractorFactory,
+) {
+
+    fun add(text: String?): Int {
+        return factory.get(text)
+            .extract(text ?: "")
+            .sum().value
```

Comment:
메서드 체이닝의 경우 한 줄에 하나의 메서드만 호출하도록하면 조금 더 일관성있게 깔끔한 코드가 될 것 같아요!
(물론 프로퍼티를 get하는 부분은 조금 애매하게 느껴지기는 하네요. 😅)
```suggestion
            .sum()
            .value
```